app/src/App.tsx

```diff
@@ -100,6 +109,15 @@ interface ItemProps {
 }
 
 function Item({ icon, title, onPress }: ItemProps) {
+  if (Platform.OS === 'macos') {
+    return (
+      <Pressable style={styles.button} onPress={onPress}>
+        {icon && <Text style={styles.title}>{icon + '  '}</Text>}
+        <Text style={styles.title}>{title}</Text>
+      </Pressable>
+    );
+  }
+
   return (
     <RectButton style={styles.button} onPress={onPress}>
```

Comment:
I don't remember what was the reason to use `RectButton` here instead of `Pressable`, can you please check it? Maybe we could use `Pressable` on all platforms.